server/test/performance/run_diffexp.py

```diff
@@ -36,8 +36,8 @@ def main():
     app_config.update_server_config(app__verbose=True)
     app_config.complete_config()
 
-    loader = MatrixDataLoader(args.dataset)
-    adaptor = loader.open(app_config)
+    loader = MatrixDataLoader(args.dataset, app_config)
```

Comment:
you could also pass the dataset_config in here, depending on which path you are trying to test

Reply:
Yes, I could pass in the dataset_config, but in this isn't so much of a test case as a script to test performance.  Since there is only the default dataset_config in this script then we can let the MatrixDataLoader get it from the app_config.